pkgs/tools/system/throttled/default.nix

```diff
@@ -11,7 +11,11 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-0MsPp6y4r/uZB2SplKV+SAiJoxIs2jgOQmQoQQ2ZKwI=";
   };
 
-  nativeBuildInputs = [ python3Packages.wrapPython ];
+  nativeBuildInputs = [
+    gobject-introspection
+    python3Packages.wrapPython
+    wrapGAppsHook3
```

Comment:
If this is not a GTK app, `wrapGAppsNoGuiHook` should be sufficient.

Reply:
see #311180